pkg/controller/whisker/controller.go

```diff
@@ -223,49 +188,32 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
-	// Guardian needs a certificate for mTLS with Goldmane.
-	// TODO: Add this to the trusted bundle. This isn't stritctly needed, since the bundle already includes the operator CA that
-	// signed this certificate. But in order to support custom user-supplied certificates, we will need to do this.
-	guardianCertificateNames := dns.GetServiceDNSNames("guardian", whisker.WhiskerNamespace, r.clusterDomain)
-	guardianCertificateNames = append(guardianCertificateNames, "localhost", "127.0.0.1")
-	guardianKeyPair, err := certificateManager.GetOrCreateKeyPair(r.cli, whisker.GuardianKeyPairSecret, whisker.WhiskerNamespace, guardianCertificateNames)
+	trustedBundle, err := certificateManager.CreateNamedTrustedBundleFromSecrets(whisker.WhiskerDeploymentName, r.cli,
+		common.OperatorNamespace(), false,
+		goldmane.GoldmaneKeyPairSecret)
```

Comment:
```suggestion
	trustedBundle, err := certificateManager.CreateNamedTrustedBundleFromSecrets(
	    whisker.WhiskerDeploymentName, 
	    r.cli,
	    common.OperatorNamespace(), 
	    false,
	    goldmane.GoldmaneKeyPairSecret,
	)
```

If we're splitting across multiple lines we gotta go all the wayyy

Reply:
I personally don't agree with it should be all on one line or every parameter should be on it's own line.

I parse large blocks of code better when it's not somewhat needlessly spread out over more lines (this adds up).

I'll go with your precendence on this, but this does not make it easier for me to read.

Reply:
Yeah I think our internal coding standards are not very clear about this. My editor will fix this for you next time I open the file though. I find inconsistent spacing to be very hard to read - two variables on one line and one variable the next and then two again? I'm going to miss something.